### ISSUE TITLE
Fix Wagtail image uploading

### ISSUE THREAD

**Issue by noahmanger** [18F/fec-cms#714]
Wagtail's built-in image uploader isn't working on production. It works for me fine locally, but not in production or staging. Could this be an issue with how the server is configured?

---

## Update
We need to configure Wagtail / Django to upload images to S3 since we cannot write to local storage. This is now issue to set up that functionality. This issue will be done when admins can upload media (whether images or other documents) to an S3 bucket and those assets can be visible on a page.

**Comment by noahmanger** [comment]
I just updated the body of this issue based on what we discussed in standup. After a little digging I found a couple resources that seem relevant:

- https://www.caktusgroup.com/blog/2014/11/10/Using-Amazon-S3-to-store-your-Django-sites-static-and-media-files/
- http://pritishc.com/blog/2015/09/06/uploading-with-django-and-amazon-s3/
- https://github.com/bradleyg/django-s3direct

No idea what the right approach is. 

Curious if maybe @jmcarp has thoughts on the matter, given what you know about Wagtail and our infrastructure.
